README.md

```diff
@@ -2,272 +2,396 @@
 
 ## Learning Goals
 
-- Create nested functions.
-- Explain that the environment in which a function is created gets added to its
-  scope chain.
-- Describe how the scope chain makes variables and functions declared in the
-  outer environment available within a nested function.
-- Explain how the JavaScript engine makes a first pass over your code before
-  executing it.
-- Describe what happens during the _execution phase_ of the JavaScript runtime.
+- Examine how scopes can be nested.
```

Comment:
Consider updating "_Examine_" to something like "_Determine_ (or _Identify_) how scopes can be nested" to make this goal even more specific and measurable.

Reply:
I like "Identify" as an option!